pom.xml

```diff
@@ -4059,6 +4059,11 @@
         <version>${version.org.jsoup}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.keycloak</groupId>
+        <artifactId>keycloak-admin-client</artifactId>
+        <version>${version.org.keycloak}</version>
+      </dependency>
```

Comment:
this is not in test scope intentionally?

Reply:
added test scope